src/components/PostPageDetail/index.js

```diff
@@ -67,9 +66,9 @@ const PostPageDetailIdComponent = (props) => {
   const [statusUpvote, setStatusUpvote] = React.useState(false);
   const [statusDownvote, setStatusDowvote] = React.useState(false);
   const [loadingPost, setLoadingPost] = React.useState(false);
-  const [item, setItem] = React.useState(null);
   const navigation = useNavigation();
   const route = useRoute();
+  const [item, setItem] = React.useState(route?.params?.data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the initialization of `item` state is a good one as it allows for the component to be initialized with data passed from the route parameters. However, this could potentially introduce a bug if `route.params.data` is undefined or not in the expected format. It would be better to add some error handling here.
```diff
- const [item, setItem] = React.useState(route?.params?.data);
+ const [item, setItem] = React.useState(route?.params?.data || {});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->